WordPress/WordPressTest/CoreDataManagerMock.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+
+@testable import WordPress
+
+class CoreDataManagerMock: CoreDataManager, ManagerMock {
+    override init() {
+        super.init()
+        ContextManager.overrideSharedInstance(self)
+    }
+
+    var _managedObjectModel: NSManagedObjectModel?
+    override var managedObjectModel: NSManagedObjectModel {
+        set {
+            _managedObjectModel = newValue
+        }
+
+        get {
+            return _managedObjectModel ?? super.managedObjectModel
+        }
+    }
+
+    var _persistentStoreCoordinator: NSPersistentStoreCoordinator?
+    override var persistentStoreCoordinator: NSPersistentStoreCoordinator {
+        set {
+            _persistentStoreCoordinator = newValue
+        }
+
+        get {
+            return _persistentStoreCoordinator ?? _persistentContainer.persistentStoreCoordinator
+        }
+    }
+
+    var _mainContext: NSManagedObjectContext?
```

Comment:
Perhaps we wanna set to private all of the internal instances? (`var _XXXX`)

Reply:
Done in 9bfcef3d2ca221bdd070318683aff57179962657